platform/pensando/sonic-platform-modules-dpu/sonic_platform/chassis.py

```diff
@@ -0,0 +1,478 @@
+# {C} Copyright 2023 AMD Systems Inc. All rights reserved
+#############################################################################
+# Pensando
+#
+# Module contains an implementation of SONiC Platform Base API and
+# provides the Chassis information which are available in the platform
+#
+#############################################################################
+
+try:
+    import os
+    import threading
+    import time
+    import sys
+    import subprocess
+    from sonic_platform_base.chassis_base import ChassisBase
+    from .helper import APIHelper
+    import syslog
+    import inspect
+except ImportError as e:
+    raise ImportError(str(e) + "- required module not found")
+
+PORT_START = 1
+PORT_END = 2
+NUM_THERMAL = 2
+NUM_VOLTAGE_SENSORS = 3
+NUM_CURRENT_SENSORS = 3
+HOST_REBOOT_CAUSE_PATH = "/host/reboot-cause/"
+REBOOT_CAUSE_FILE = "reboot-cause.txt"
+HOST_CHK_CMD = "docker > /dev/null 2>&1"
+REBOOT_CAUSE_SOFTWARE = "Software causes"
+REBOOT_CAUSE_EXTERNAL = "External causes"
+RESET_CAUSE_PATH = "/sys/firmware/pensando/rstcause/this_cause"
+
+SYSLOG_IDENTIFIER = "sonic_platform.chassis"
+
+#cpld masks for system led
+SYSTEM_LED_GREEN   = 0x7
+SYSTEM_LED_YELLOW  = 0x38
+SYSTEM_LED_REG     = 0x15
+
+def log_info(msg):
+    syslog.openlog(SYSLOG_IDENTIFIER)
+    syslog.syslog(syslog.LOG_INFO,msg)
+    syslog.closelog()
+    # print(msg)
+
+def log_emerg(msg):
+    syslog.openlog(SYSLOG_IDENTIFIER)
+    syslog.syslog(syslog.LOG_EMERG,msg)
+    syslog.closelog()
+    # print(msg)
+
+def log_warning(msg):
+    syslog.openlog(SYSLOG_IDENTIFIER)
+    syslog.syslog(syslog.LOG_WARNING,msg)
+    syslog.closelog()
+    # print(msg)
```

Comment:
@shanshri please use SysLogger https://github.com/sonic-net/sonic-buildimage/blob/master/src/sonic-py-common/sonic_py_common/syslogger.py

Reply:
@prgeor, used SysLogger for logging.